client/src/lib/dependency-injection/graphql/buildDeleteMutation.js

```diff
@@ -1,9 +1,9 @@
 import { defaultTag } from './tags';
-import { getSingularName } from './helpers';
+import { getSingularName, getIdOnlyVariables, getIdOnlyParams } from './helpers';
 
 const buildDeleteMutation = (tag = defaultTag) => (
-  tag`mutation Delete${getSingularName}($IDs:[ID]!) {
-    delete${getSingularName}(IDs: $IDs)
+  tag`mutation Delete${getSingularName}${getIdOnlyVariables} {
+    delete${getSingularName}${getIdOnlyParams}
   }`
 );
```

Comment:
Turning `$IDs:[ID]!` and `IDs: $IDs` into default fallbacks, which can be overridden by setting explicit params. See the new related test for an example.